docs/design/coreclr/jit/Jit Stack Security.md

```diff
@@ -0,0 +1,64 @@
+# .NET Stack Security
+
+This document describes mechanisms in the .NET code generator to ensure stack security at runtime.
```

Comment:
Is "security" the right term here? Most other places call it buffer overflow protection or stack cookies, specifically because it isn't really a "security feature". You require a shadow stack or similar for "proper" security and otherwise its largely just a mitigation for common types of overflow mistakes.